README.md

```diff
@@ -19,48 +21,27 @@ A Visual Studio Code test extension that allows users to run their [Kani Rust Ve
 ## Requirements
 
 -   Visual Studio Code 1.50 or newer
--   [Kani](https://github.com/model-checking/kani) 0.12 or newer
-
-## Installation
+-   [Kani](https://github.com/model-checking/kani) 0.29 or newer
 
-This extension is still in beta and has not yet been published on the marketplace. If you wish to use it now, you can [download the binary](https://github.com/model-checking/kani-vscode-extension) and install it manually.
+## Extension Settings
 
-### Install through the Binary
+| Setting                           | Description                                                                                                                                                                                        | Default                                                        |
+| :-------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------- |
+| `kani.enable-codelens` | Enable Codelens actions for `Run Test (kani)` & `Debug Test (kani)`.                                                                                                      | `true`                                                         |
+| `kani.show-output-window`     | Optional string to control when the document should be verified (choose between `true` or `false`.                                                                                 | `false`                                                     |
```

Comment:
Is ``(choose between `true` or `false` needed here``? Also, I'm not sure what "document" is referring to here.